packages/ui/src/app/pages/Members/Members.tsx

```diff
@@ -27,10 +27,10 @@ export const Members = () => {
 
   const { members, isLoading, totalCount, pageCount } = useMembers({ order, filter, page })
 
-  const roles: { [key: string]: string } = {}
+  const roles = new Set<MemberRole>()
   for (const member of members) {
     for (const role of member.roles) {
-      roles[memberRoleTitle(role)] = memberRoleAbbreviation(role)
+      roles.add(role)
     }
   }
```

Comment:
I think that we need some sort of map/configuration of possible roles. For now, we have Group + Worker/Leader. So you can assume that the list is static.

However, it might be too soon to implement a query for it, so don't worry about that. We're waiting for the schema upgrade anyway so we can add that later.

Reply:
Let's have this done in https://github.com/Joystream/pioneer/issues/757#issuecomment-856535966.